recipe/meta.yaml

```diff
@@ -38,20 +35,14 @@ outputs:
         - python
         - pip >=9.0.1
         - certifi >=2020.06.20
-        # freetype.pc requires zlib.pc
         - freetype
-        - zlib
```

Comment:
please add here something like '1.2' as version for zlib

Reply:
zlib has been actually removed from the recipe as matplotlib does not use it as direct dependency.